charts/server/values.yaml

```diff
@@ -164,7 +192,6 @@ dnsConfig: {}
 
 # -- Enabled internal metrics endpoint.
 metrics:
+  # -- enable metrics in woodpecker
   enabled: false
   port: 9001
-  interval: 60s
-  path: /metrics
```

Comment:
I'm not sure if WP supports changing the metrics path but in any case, we could leave the param as is? (Just to have less breaking changes)

Reply:
if the application change it, the helmchart should be change also (and no user of the helmchart).

or do you plan to make the path it configurable in the woodpecker-server itself

Reply:
but yes, we could let **interval** still be there.
(i personal prefer a clean seperation of metric enabled and usage of a PodMonitor).

so if you like it there i revert it

Reply:
> or do you plan to make the path it configurable in the woodpecker-server itself

I don't think so but I am just trying to ask "smart" questions :) I also didn't put the value there initially.

> (i personal prefer a clean seperation of metric enabled and usage of a PodMonitor).

Clean separations are always good but if this is resulting in a breaking change anyhow, we could also make more "breaking" changes.

I'll let it to you where to place `interval` - but if you move it, please document it -> i.e. we need some upgrade notes for this PR anyhow :)

Reply:
where, in which file should i document that breaking change?